src/database/migrations/0029-create-base_field_localizations.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE base_field_localizations (
+  base_field_id INTEGER NOT NULL,
+  language VARCHAR NOT NULL,
+  label VARCHAR NOT NULL,
+  description VARCHAR,
+  created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+  PRIMARY KEY (base_field_id, language),
+  FOREIGN KEY (base_field_id) REFERENCES base_fields(id)
```

Comment:
One of the concerns @hminsky2002 and I talked about the other day is that this schema allows base fields without _any_ localizations, which is currently not possible. There are some more complicated technical controls we could put into place. I think we landed on not worrying about that yet, as this PR reflects.

After talking with @kfogel today, given that creating or modifying base fields is restricted to administrators, he and I agreed that relying on social controls ("always enter at least one translation when creating a new base field") is sufficient here.

So, all of this to say: :+1: !